src/components/Publish/_validation.ts

```diff
@@ -54,13 +55,23 @@ const validationService = {
   })
 }
 
+const decimalDigitsValidation = new RegExp(/^\d+(\.\d{1,6})?$/)
```

Comment:
hard to come up with clear variable name so we don't have to explain regex in code comment. Is `maxDecimalsValidation` more clear?